backend/tests/consts.py

```diff
@@ -25,3 +25,5 @@
 TRANSFORM_CORRECT_INPUT_CSV = (
     pathlib.Path(__file__).parent / "test_files" / "transform" / "correct_input.csv"
 )
+
+EXPORT_FORM_DICT_NAME = "data"
```

Comment:
Well, i think it's used in much more places, that only one. Isn't it?

Reply:
Also it should be named `EXPORT_FORM_DATA_KEY` or something like this. What do you think?